packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol

```diff
@@ -176,6 +177,8 @@ contract LongShortPair is Testable, Lockable {
     /**
      * @notice Return a pair of long and short tokens equal in number to tokensToCreate. Returns the commensurate amount
      * of collateral to the caller for the pair of tokens, defined by the collateralPerPair value.
+     * @dev The caller does not need to approve this contract to transfer any amount of `tokensToRedeem` since long
```

Comment:
Could you also add that the LSP contract has the burner role?

Reply:
Done